WooCommerce/WooCommerceTests/AppCoordinatorTests.swift

```diff
@@ -305,6 +388,7 @@ private extension AppCoordinatorTests {
     /// Convenience method to make AppCoordinator instances.
     func makeCoordinator(window: UIWindow? = nil,
                          stores: StoresManager? = nil,
+                         storageManager: StorageManagerType? = nil,
```

Comment:
nit: since `storageManager` is already initialized for all test cases and the same value is injected to `AppCoordinator` below, maybe this parameter isn't needed?

Reply:
Updated in 392eb668c35b4bad54e8e8dc1e2ab3b0b9a4c868.